### ISSUE TITLE
Email not sent

### ISSUE THREAD

**Issue by vjsablok1991** [sierracircle/services-checker#4]
the script did restart the service, however, it did not send any email notification

**Comment by sierracircle** [comment]
you need a mail sender set up on your server. for example sendmail 

`sudo apt-get install sendmail`

**Comment by vjsablok1991** [comment]
@sierracircle sendmail is already there, however, it is still not sending any email

**Comment by sierracircle** [comment]
Can you test sendmail from the command line? replace your@email.com  with your email.

`echo "Subject: test" | sendmail -v your@email.com`

**Comment by vjsablok1991** [comment]
@sierracircle i did, the console doesn't give any error message, it says successful, however, the mail has not arrived yet. I have tried this multiple times
i tried the same command from local, it was working.
What might be the issue with this?

**Comment by sierracircle** [comment]
Hi @vjsablok1991   most likely this is a sendmail issue rather than an issue with the script.  but here are a few things you can try:

- Have you checked your spam mail folder? 
- Have you tried sending to a different email address (if your email is gmail, try a yahoo address or hotmail or some other server)
- have you checked your firewall settings? You can temporarily disable your firewall with this:

```
sudo iptables -P INPUT ACCEPT
sudo iptables -P OUTPUT ACCEPT
sudo iptables -F
```

check your sendmail again, then restore your firewall settings with this (or just reboot your server):
`    sudo netfilter-persistent reload`
